cmd/elrond/ring.go

```diff
@@ -346,8 +383,8 @@ var ringListCmd = &cobra.Command{
 					strconv.Itoa(ring.Priority),
 					strings.Join(igs, "\n"),
 					strconv.Itoa(ring.SoakTime),
-					ring.Image,
-					ring.Version,
+					ring.ActiveReleaseID,
+					ring.DesiredReleaseID,
```

Comment:
As a bit more user friendly approach you could consider fetching those releases and displaying versions.

Reply:
Thanks, I did some improvements there plus added indexing for Image, Version and Force in ring release tables to avoid generating new release entries everytime. Now during a release we always check first if a release with these details already exists and if not we create it.